# Source: Internal
# Info: Written internally so this component can run its testsuite on Solaris.
# Status: This patch is offered at https://bugs.launchpad.net/ubuntu/+source/sbsigntool/+bug/1588548

--- ORIGINAL/tests/test-wrapper.sh	2016-06-10 15:01:05.749653696 -0700
+++ sbsigntool-0.6/tests/test-wrapper.sh	2016-06-14 17:08:53.739736437 -0700
@@ -16,7 +16,7 @@
 
 # 'test' needs to be an absolute path, as we will cd to a temporary
 # directory before running the test
-test="$PWD/$1"
+test="$1"
 rc=0
 
 function run_test()
